README.md

```diff
@@ -1 +1,19 @@
 # @comet/upgrade
+
+## Usage
+
+```bash
+npx @comet/upgrade <target-version>
+```
+
+For example
+
+```bash
+npx @comet/upgrade v4
+```
+
+or
+
+```bash
+npx @comet/upgrade 4
```

Comment:
while it is not bad to support 4 and v4, I would not document both

Reply:
> while it is not bad to support 4 and v4, I would not document both.

I've removed the example for 4

> maybe document how to use with a local working copy of comet-upgrade? (for development)

I've added some instructions. However, there is an issue with Node v14: I've tried globally linking the package using `npm link` or `npm install -g`, so that I may used it in another project (using `npx @comet/upgrade`) for testing. This doesn't work in Node v14, but works in v16 and v18 (probably due to new NPM version). Should we use a newer Node version for this project? Otherwise one would have to use `bin/index.js 4` for testing, which only works in the local repository (or in other projects by using absolute file system paths).

Reply:
using bin/index (for development) is what I expected and perfectly fine. The other might even be confusing.